sdk/identity/identity/src/msal/utils.ts

```diff
@@ -74,11 +74,15 @@ export function getAuthority(tenantId: string, host?: string): string {
  * by sending it within the known authorities in the MSAL configuration.
  * @internal
  */
-export function getKnownAuthorities(tenantId: string, authorityHost: string): string[] {
+export function getKnownAuthorities(tenantId: string, authorityHost: string, knownAuthorities: string[]|undefined): string[] {
```

Comment:
Shouldn't this be knownAuthorities?: string[] ?

Reply:
Yes, agree